ts/features/itwallet/issuance/screens/ItwIssuanceEidPreviewScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import { EidCardPreview } from "../../common/components/EidCardPreview";
+import { ItwCredentialClaimsList } from "../../common/components/ItwCredentialClaimList";
+import {
+  ItWalletError,
+  getItwGenericMappedError
+} from "../../common/utils/itwErrorsUtils";
+import { ItwCredentialsMocks } from "../../common/utils/itwMocksUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+
+export const ItwIssuanceEidPreviewScreen = () => {
+  const navigation = useIONavigation();
+  const eidOption = O.some(ItwCredentialsMocks.eid);
+
+  /**
+   * Renders the content of the screen if the PID is decoded.
+   * @param eid - the decoded eID
+   */
+  const ContentView = ({ eid }: { eid: StoredCredential }) => {
+    React.useLayoutEffect(() => {
+      navigation.setOptions({
+        headerShown: true
+      });
+    }, []);
+
+    return (
+      <IOScrollViewWithLargeHeader
+        title={{
+          label: I18n.t("features.itWallet.issuance.credentialPreview.title", {
+            credential: eid.displayData.title
+          })
+        }}
+      >
+        <View style={styles.preview}>
+          <EidCardPreview />
+        </View>
+        <View style={styles.dropShadow}>
+          <VSpacer size={24} />
+        </View>
+        <View style={styles.content}>
+          <ItwCredentialClaimsList data={eid} />
+        </View>
+        <ContentWrapper>
+          <VSpacer size={32} />
+          <ButtonSolid
+            label={I18n.t(
+              "features.itWallet.issuance.credentialPreview.actions.primary"
+            )}
+            icon="add"
+            iconPosition="end"
+            onPress={() => undefined}
+            fullWidth={true}
+          />
+          <VSpacer size={24} />
+          <View style={{ alignSelf: "center" }}>
+            <ButtonLink
+              label={I18n.t(
+                "features.itWallet.issuance.credentialPreview.actions.secondary"
+              )}
+              onPress={() => undefined}
+            />
+          </View>
+        </ContentWrapper>
+      </IOScrollViewWithLargeHeader>
+    );
+  };
+
+  /**
+   * Error view component which currently displays a generic error.
+   * @param error - optional ItWalletError to be displayed.
+   */
+  const ErrorView = ({ error: _ }: { error?: ItWalletError }) => {
+    const mappedError = getItwGenericMappedError(() => navigation.goBack());
+    return <OperationResultScreenContent {...mappedError} />;
+  };
+
+  return pipe(
+    eidOption,
+    O.fold(
+      () => <ErrorView />,
+      eid => <ContentView eid={eid} />
+    )
+  );
+};
+
+const styles = StyleSheet.create({
+  preview: {
+    paddingHorizontal: 24
+  },
+  dropShadow: {
+    backgroundColor: IOColors.white,
+    shadowColor: IOColors.black,
+    shadowOffset: { width: 0, height: -4 },
+    shadowRadius: 20,
+    shadowOpacity: 0.2,
+    elevation: 5
+  },
+  content: {
+    flex: 1,
+    backgroundColor: IOColors.white, // Add background to cover shadow
```

Comment:
nice trick :-)

Reply:
<img src="https://en.meming.world/images/en/4/4a/Modern_Problems_Require_Modern_Solutions.jpg" width="300" />

Reply:
@mastro993 What about using the `appBackground-primary` theme key, instead? Yes, I know the dark mode is not our top priority, but I'm a hopeless optimist

Reply:
> @mastro993 What about using the `appBackground-primary` theme key, instead? Yes, I know the dark mode is not our top priority, but I'm a hopeless optimist

Added in https://github.com/pagopa/io-app/pull/5780/commits/7f4c5942cfd7beb05b55d7c5f4412e5631c92c5a